[ACM-6607]: Indicate that hosted clusters are automatically imported (#5428)
open-cluster-management/rhacm-docs: clusters/hosted_control_planes/importing_hosted_cluster_aws.adoc

[#importing-hosted-cluster-aws]
= Importing a hosted control plane cluster on AWS

> Hosted clusters are automatically imported into {mce-short} after the hosted control plane becomes available. The following steps describe how to import hosted clusters manually.

Reviewer: ```suggestion
Hosted clusters are automatically imported into {mce-short} after the hosted control plane becomes available. The following steps describe how to import hosted clusters manually:
```

> Hosted clusters are automatically imported into {mce-short} after the hosted control plane becomes available. The following steps describe how to import hosted clusters manually.

Reviewer 2: My main change request is to add the colon after "manually", but here is my complete suggestion: 

```suggestion
Hosted clusters are automatically imported into {mce-short} after the hosted control plane becomes available. If you want to import hosted clusters manually, complete the following steps:
```

Author: Great suggestion!
